tests/test/test.js

```diff
@@ -92,12 +92,72 @@ assert.equalEpsilon = function(actual, expected) {
     if (typeof actual !== 'number') {
         throw new Error("Expected " + actual + " to be a number.");
     }
-    let epsilon = Math.abs(expected * 1e-6);
+    let epsilon = 1e-6;
+    epsilon = Math.max(epsilon, Math.abs(expected * epsilon));
     if (Math.abs(actual - expected) > epsilon) {
         throw new Error("Expected " + actual + " to equal " + expected);
     }
 };
 
+assert.equalEpsilonArray = function(actual, expected) {
+    const length = expected.length;
+    assert.strictEqual(actual.length, length);
+
+    for (let i = 0; i < length; ++i) {
+        assert.equalEpsilon(actual[i], expected[i]);
+    }
+};
+
+function getAccessorData(gltfPath, asset, accessorIndex, bufferCache) {
+    // This is only for testing exporter output, it does not handle enough
+    // cases to parse arbitrary glTF files from outside this test suite.
```

Comment:
More idle curiosity than an actual suggestion, but does glTF-Pipeline have a programmatic API we could be using here? I.e. import it from NPM, use it to parse the file, and provide some accessor data?

I've written some code of my own along these lines, which we could also use here, but glTF-Pipeline is certainly the more mature option.

```js
import { NodeIO } from '@gltf-transform/core';

const io = new NodeIO(fs, path);
const container = io.read('test.gltf');
const array = container.getAccessorArray( accessorIndex ); // TypedArray
```

Reply:
I just asked someone who still part-time maintains gltf-pipeline, sounds like there's no public API for this, unfortunately.  There's internal API for it, not exposed.

But for the short term I think our tests should be able to get pretty far with a cheap decoder.  The bigger problem for us is likely to be the actual vertex order (or morph target order) being unstable.  In this particular test case, two morph targets have the same key/value data, so having the order swap makes no difference in the output.

Reply:
I'm not opposed to using your gltf-transform thing, if you think that's mature enough to use here (certainly more mature than what I've written in this PR, I would imagine)